dashboard/app/controllers/datasets_controller.rb

```diff
@@ -0,0 +1,18 @@
+class DatasetsController < ApplicationController
+  before_action :require_levelbuilder_mode
+  before_action :initialize_firebase
+  authorize_resource class: false
+
+  def show_manifest
```

Comment:
nit: add comments to each route that describe the URL (e.g., this one would be `# GET /datasets/manifest`)